src/rust/bitbox02-sys/build.rs

```diff
@@ -109,16 +116,37 @@ pub fn main() -> Result<(), &'static str> {
         .args(["--allowlist-function", "memory_is_initialized"])
```

Comment:
Here it would be nice to exclude the whole statement from formatting, so there is one line per arg. Much easier to read/search/modify that way .

Reply:
I refactored it